<!-- -->
<launch>

	
	<node name="grid_tracking_node" pkg="grid_tracking" type="grid_tracking_node" output="screen">
		<param name="file_path" value="$(find grid_tracking)/vertex_file/data.txt"/>
		<param name="disThreshold" value="3.0"/>
		<param name="tracking_speed" value="2.0"/>
    </node>
      
</launch>
